frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -125,25 +131,33 @@ export const createTopologyNodeData = (
   const deploymentsLabels = _.get(deploymentConfig, 'metadata.labels', {});
   const deploymentsAnnotations = _.get(deploymentConfig, 'metadata.annotations', {});
   const nodeResourceKind = _.get(deploymentConfig, 'metadata.ownerReferences[0].kind');
+  const operatorBackedService = nodeResourceKind in operatorBackedServiceKindMap;
+  const getNodeIcon = () => {
```

Comment:
Any reason this was made a function? Seems like it's only used once and lazy computing isn't helping in this case.

Reply:
No specific reason for making it a function. Just didn't want to write a big ternary operator expression.

Reply:
I don't think you need to go to either extreme... you can just create the variables inline (instead of wrapped in a function) and just use the result from the `return` in the spot where you call it now.

Reply:
Updated.